setup.cfg

```diff
@@ -122,7 +126,7 @@ addopts =
     --verbose
 #    In order to use xdist, the developer can add, for example, the following
 #    arguments:
-#    --dist=load --numprocesses=auto
```

Comment:
I can no longer find docs for this option.

Reply:
Thanks for checking. It's good to remove clutter like this then.